tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2358,6 +2357,23 @@ private boolean pathHasError(String localeID, String xpath) {
                 }
                 return false;
             }
+
+            @Override
+            public
+            void handleEnd() {
+                // look for paths in vxmlFile that aren't in baselineFileUnresolved
+                // TODO: another optimal way?
+                final Set<String> vPaths = new HashSet<>();
+                final Set<String> bPaths = new HashSet<>();
+                vxmlFile.getPaths("", null, vPaths);
+                baselineFileUnresolved.getPaths("", null, vPaths);
```

Comment:
Looks like this should read into bPaths, not vPaths.

Reply:
Otherwise code looks good.

(a more optimal way to do it would be to get the vPaths in the handleStart() as a class parameter. Then with each handlePath() is called, drop the path from vPath. However, probably not worth the effort.

Reply:
BTW, the ERR paths are covered by the personnames inclusion, so you probably won't see a difference.

Reply:
> Looks like this should read into bPaths, not vPaths.

Whoops good catch. 

That would explain what I saw.